index.js

```diff
@@ -5,7 +5,19 @@ const morseCodeDictionary = require("./morse-code-dictionary.json");
  * @param {String[]}  - An array of strings.
  * @returns {string[]} An array of strings.
  */
-function sortByStringLength() {}
+function sortByStringLength(words) {
+  if (!Array.isArray(words)) {
```

Comment:
Nice work here with the checking if words is in fact an array.  

Any other ways we can do this?